CHANGELOG.md

```diff
@@ -1086,7 +1086,7 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Replace references to `country_delivery_partners` with `country_partner_organisations`, including database migration
 - Replace code references to `uk_dp_named_contact` with `uk_po_named_contact`, including database migration
 - Remove "PO Definition" from "Aims/Objectives" CSV headers
-- Add ability for BEIS users to create and update Level B activities in bulk by uploading a CSV 
+- Add ability for BEIS users to create and update Level B activities in bulk by uploading a CSV
```

Comment:
Might make more sense to include this trailing space removal in a commit that actually updates the changelog (if it all), but that said, should we include a line for this since it's now beyond the scope of just bulk uploads, e.g. "Make partner organisation identifier optional for Level B activities (both via the individual new activity form and bulk upload)"

Reply:
Yeah, I think it'd be good to include this as a new changelog item as it's changing the UI journey too.